contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -30,6 +29,10 @@ abstract contract VaultHub is PausableUntilWithRoles {
         mapping(address => uint256) vaultIndex;
         /// @notice allowed beacon addresses
         mapping(bytes32 => bool) vaultProxyCodehash;
+        /// @notice maximum number of vaults that can be connected to the hub
+        uint256 maxVaultsCount;
+        /// @notice maximum size of the single vault relative to Lido TVL in basis points
```

Comment:
`size` is ambiguous! what does it mean?
- share of shares
- valuation ratio against the protocol's TVL
- amount of ether
etc.